sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -266,14 +261,36 @@ public partial class DownloadManifestOptions
     {
         public DownloadManifestOptions(string tag = null, string digest = null) { }
         public string Digest { get { throw null; } }
+        public Azure.Containers.ContainerRegistry.Specialized.ManifestMediaType? MediaType { get { throw null; } set { } }
         public string Tag { get { throw null; } }
     }
     public partial class DownloadManifestResult
     {
         internal DownloadManifestResult() { }
         public System.BinaryData Content { get { throw null; } }
         public string Digest { get { throw null; } }
-        public Azure.Containers.ContainerRegistry.Specialized.ArtifactManifest Manifest { get { throw null; } }
+        public Azure.Containers.ContainerRegistry.Specialized.ManifestMediaType MediaType { get { throw null; } }
+        public Azure.Containers.ContainerRegistry.Specialized.OciManifest AsOciManifest() { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ManifestMediaType : System.IEquatable<Azure.Containers.ContainerRegistry.Specialized.ManifestMediaType>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public static readonly Azure.Containers.ContainerRegistry.Specialized.ManifestMediaType DockerManifestList;
```

Comment:
should we define it if we're not going to support it?

Reply:
Yeah, I went back and forth on this when I wrote it -- glad you thought to comment on it.

If we plan to add support for types later, I like having the extensible enum for convenience.  But it also feels weird to ship an enum with only one value in it, so I added the other known values to keep it company.

Since we could have both DockerV2 and OciManifest in the enum, though, and leave lists out until we properly support them, maybe we could move forward with that instead.  I agree with your point that it's strange to provide the option for something we don't currently support.

Reply:
Addressed.